components/AppFooter/AppFooter.tsx

```diff
@@ -0,0 +1,164 @@
+import { createStyles, Text, Container, ActionIcon, Group } from '@mantine/core';
+import AppLink from '../AppLink/AppLink';
+import Image from "next/image";
+import LogoSmall from "../../public/logo_small.png";
+import { DiscordIcon } from "../AppHeader/DiscordIcon";
+import { SteamIcon } from "./SteamIcon";
+import { FooterTextLinks } from "./FooterTextLinks"
+
+const useStyles = createStyles((theme) => ({
```

Comment:
You can move the styles to `AppFooter.styles.ts` to have the same structure as other components (See AppHeader/HeaderControl)